server/graphql/v1/mutations/expenses.js

```diff
@@ -671,6 +672,20 @@ export async function payExpense(req, args) {
     );
   }
 
+  // If the amount of the expense is larger than $1000 enforce 2FA if the user has it turned on
+  const acceptedPayoutMethods =
+    payoutMethodType === PayoutMethodTypes.PAYPAL || payoutMethodType === PayoutMethodTypes.BANK_ACCOUNT;
+  const hostHasPayout2FAEnabled = get(host, 'settings.payoutsTwoFactorAuth.enabled', null);
```

Comment:
That will not change the behavior, but default should be a boolean here,  `false`.